client/blocks/comments/post-trackback.tsx

```diff
@@ -0,0 +1,70 @@
+import config from '@automattic/calypso-config';
+import { Gridicon } from '@automattic/components';
+import { get } from 'lodash';
+import TimeSince from 'calypso/components/time-since';
+
+import './post-comment.scss'; // yes, this is intentional. they share styles.
+
+interface PostTrackbackProps {
+	commentId: number;
+	commentsTree: Record< number, PostComment >;
+}
+
+interface PostComment {
+	data: {
+		author: {
+			ID: number;
+			name: string;
+			URL: string;
+		};
+		date: string;
+		URL: string;
+	};
+}
+
+function unescape( str: string ): string {
+	return str.replace( /&#(\d+);/g, ( match, entity ) => String.fromCharCode( entity ) );
+}
+
+export default function PostTrackback( props: PostTrackbackProps ): JSX.Element | null {
+	const commentsTree = props.commentsTree;
+	const comment = get( commentsTree[ props.commentId ], 'data' );
+	if ( ! comment ) {
+		return null;
+	}
+	const unescapedAuthorName = unescape( get( comment, 'author.name', '' ) );
+
+	const authorUrlLink =
+		config.isEnabled( 'reader/user-profile' ) && comment.author?.ID
+			? `/read/users/${ comment.author.ID }`
+			: comment.author?.URL;
```

Comment:
Can we assume that if a comment is a trackback, that the URL will be present in the `author` object and that we won't need to look it up via the site ID?

Reply:
Where are we looking it up via site ID?

Reply:
Right here in the post-comment component: https://github.com/Automattic/wp-calypso/blob/2f46423b220f9ceb55b89d917d3b27842dbd76ab/client/blocks/comments/post-comment.jsx#L319

Reply:
Ah never mind, I'm reading this wrong. All authors should have an ID and when the config flag is enabled, the other case is of no concern,